scripts/setup_script_unit.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up some stuff"
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+mkdir bin artifacts
+# Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
+# for eg go get ginkgo later on will produce ginkgo binary in GOBIN
+export GOBIN="`pwd`/bin"
+# Set kubeconfig to current dir. This ensures no clashes with other test runs
+export KUBECONFIG="`pwd`/config"
+export ARTIFACT_DIR="`pwd`/artifacts"
```

Comment:
I think we should not overwrite the default `ARTIFACT_DIR` on prow as per our last conversation with platform team. I think you mean `ARTIFACTS_DIR`, no? Same as here https://github.com/openshift/odo/blob/master/scripts/run_all_tests.sh#L20